### PR TITLE
Search Playlists

### PR DESCRIPTION
**Pull request by callil**
Type to search will filter results.

Removed pagination and added loading state.

Styling tweaks to match are.na look and feel.

Also reversed order of playlists list -- (but not song order)

### CODE REVIEW

File: src/containers/Playlist.js [hunk 1]
```diff
@@ -13,17 +13,31 @@ class Playlist extends Component {
...
+      } = this.props
+
+      let status = "public"
+      switch (playlist.status){
+        case "public":
+          status = "public"
+          break;
+        case "closed":
+          status = "closed"
+        break;
```
Comment (broskoski) [inline src/containers/Playlist.js:30]: Indent pls!

File: src/containers/Playlist.js [hunk 2]
```diff
@@ -35,7 +49,7 @@ class Playlist extends Component {
     const { currentOpenPlaylist, isCurrentPlaylistLoaded} = this.props
     if (isCurrentPlaylistLoaded) {
       return (
-        <div className='w-100 min-vh-100'>
+        <div className={'w-100 min-vh-100 '}>
```
Comment (broskoski) [inline src/containers/Playlist.js:52]: #nitpick don't think this has to be interpolated

File: src/containers/Playlists.js [hunk 3]
```diff
@@ -5,16 +5,55 @@ import { decode } from 'he'
...
+
   componentDidMount() {
     this.props.returnFullRoute(this.props.computedMatch.path)
   }
 
+  componentWillReceiveProps (nextProps) {
+    if (nextProps.playlists){
+      let playlists = []
+      // we can make a sort by toggle here with the reversed prop eventually
+      this.state.reversedOrder ? playlists = nextProps.playlists.reverse() : playlists = nextProps.playlists
```
Comment (broskoski) [inline src/containers/Playlists.js:25]: 👍

File: src/containers/Playlists.js [hunk 4]
```diff
@@ -5,16 +5,55 @@ import { decode } from 'he'
...
+      this.setState({playlists: playlists, initialPlaylists: playlists})
+    }
+  }
+
+  filterList = (event) => {
+    var updatedList = this.state.initialPlaylists;
+    updatedList = updatedList.filter(function (item) {
```
Comment (broskoski) [inline src/containers/Playlists.js:32]: This is fine, but we can also use the API to search the channel. #forafuturepr

File: src/containers/Playlist.js (hunk 2 above)
Comment (g-a-v-i-n) [reply src/containers/Playlist.js:52]: brackets are necessary if `'` is used instead of `"`

**Comment by broskoski** [comment]
LGTM, lets do this

File: src/containers/Playlists.js [hunk 5]
```diff
@@ -5,16 +5,55 @@ import { decode } from 'he'
 import LinkItem from '../components/LinkItem'
 
 class Playlists extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
```
Comment (g-a-v-i-n) [inline src/containers/Playlists.js:10]: Eventually I want to highlight the currently playing playlist in the '/' route. This means that there will need to be a consistant way to set the index of this playlist through search returns and pagination.

**Review by g-a-v-i-n — APPROVED** [review]

**Pull request merged**